lib/utils/utils.js

```diff
@@ -64,8 +66,50 @@ function setConsoleLogPrefix(prefix) {
     };
 }
 
+// This should be roughly similar to ARG_MAX, some systems is 131072 bytes
+// but we're going to be very conservative and cap the total size much lower than that to be safe.
+// POSIX suggestion is to go 2048 lower than the ARG_MAX value, as per https://www.in-ulm.de/~mascheck/various/argmax/
+const CMD_SIZE_LIMIT = 131072;
+const VAR_PATH = "./in/vars";
```

Comment:
Just need to ensure we don't add subfolders with name `vars` under `./in` otherwise. I don't think we do right now, but we should document. In case we e.g. unzip zip source files under `./in` and they contain a `var` folder.

See also https://github.com/adobe/asset-compute-sdk/blob/master/lib/prepare.js#L29-L34 (maybe good place to add a note about `vars` for now at least)

If we think there could be conflicts, we could have an `./var` or `./tmp` base folder (with e.g. `vars` inside, but open to more temp stuff) under the `<activationId>` folder.

Reply:
Yes I agree we want to document this new behavior so folks know what to expect.  However, rather than expect people to unzip in the /in folder we should recommend against that.  A common directory is a lot like a global namespace, and therefore if they want to unzip into a folder without stepping on other files, they should create their own subfolder.

Reply:
I moved the location to ./vars to prevent conflict and updated docs.